src/CMakeLists.txt

```diff
@@ -225,6 +225,72 @@ if (ENABLE_JIT_PROFILING)
     target_link_libraries(core PRIVATE "${VTUNE_LIBRARY}")
 endif()
 
+# --- RetroAchievements ---
+message(STATUS "Adding RetroAchievements integration sources")
```

Comment:
There's no option to disable RetroAchievements support in a build, is there? That's not great, as it's a fairly large dependency.